build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -267,7 +267,7 @@ $_baseUrlScript
         return dart.getSourceMap(module);
       });
   }
-  if (window.postMessage) {
+  if (typeof Window === 'function') {
```

Comment:
synced offline - this feels fragile to me. I think we should do some research to see if there is a canonical way to determine if we're the rendering window or a worker.